bool isFirstControllerCall
FlyingVehicleState ownCrazyflie
FlyingVehicleState[] otherCrazyflies
---
ControlCommand controlOutput